spec/views/posts_index_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+RSpec.describe '#UserIndex', type: :feature do
+  before(:each) do
+    @users = [
+      @user1 = User.create(
+        name: 'Derrick',
+        photo: 'https://img.freepik.com/free-vector/mysterious-mafia-man-smoking-cigarette_52683-34828.jpg?w=1480&t=st=1674661161~exp=1674661761~hmac=e529680000ea966150e5ea3b38241a0d8c9582faf23dac8af61ce1785cf27838',
+        bio: 'Fullstack Web developer from Uganda',
+        posts_counter: 9
+      ),
+      @user2 = User.create(
+        name: 'Lilly',
+        photo: 'https://img.freepik.com/free-vector/mysterious-mafia-man-smoking-cigarette_52683-34828.jpg?w=1480&t=st=1674661161~exp=1674661761~hmac=e529680000ea966150e5ea3b38241a0d8c9582faf23dac8af61ce1785cf27838',
+        bio: 'Teacher from Poland.', posts_counter: 1
+      )
+    ]
+
+    @posts = [
+      @post1 = Post.create(
+        author: @user1, title: 'List of all posts', text: 'This is my first post', comments_counter: 0, likes_counter: 0
+      ),
+      @post2 = Post.create(
+        author: @user1, title: 'Football', text: 'This is my second post', comments_counter: 0, likes_counter: 0
+      ),
+      @post3 = Post.create(
+        author: @user1, title: 'Current Affairs', text: 'This is my third post', comments_counter: 0, likes_counter: 0
+      )
+    ]
+
```

Comment:
- [ ] Again, let's ensure to provide the following test scenarios for the _post index view_:
   * I can see a section for pagination if there are more posts than fit on the view.
   * When I click on a post, it redirects me to that post's show page.